stdlib/public/Observation/Sources/Observation/Macros.swift

```diff
@@ -0,0 +1,26 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+#if SWIFT_OBSERVATION_MACROS
+#if compiler(>=5.8) && hasAttribute(attached)
+@available(SwiftStdlib 5.9, *)
+@attached(member)
```

Comment:
This is going to need to document the names it produces, per [SE-0389](https://github.com/apple/swift-evolution/blob/main/proposals/0389-attached-macros.md)

Reply:
what specifically needs to happen there? The `MemberMacro` will add in a field `_registrar`, the functions `transactions(for:isolation:)` and `changes(for:)`, a sub-type `_Storage`, and a field `_storage`; are those what need to be listed?

Reply:
I think your attribute should read

```swift
@attached(member, names: named(_registrar), named(transactions), named(changes), named(_Storage), named(_storage))
```